ui/src/router/components/form/components/router_config/rule_card/RuleCard.js

```diff
@@ -81,6 +82,21 @@ export const RuleCard = ({
         </EuiFlexItem>
       </EuiFlexGroup>
 
+      <EuiFormRow
```

Comment:
A screenshot of this, in the PR description would be nice, for illustration purposes.

Reply:
Missed this earlier - we also need to update the traffic rule schema used for yup validation (`ui/src/router/components/form/validation/schema.js`).

Reply:
Added validation and screenshot in PR description.